### DISCUSSION TITLE
Equivalent of the start goal from the docker-maven-plugin

### DISCUSSION THREAD

**Discussion by mpiggott** [eclipse/jkube#1773] [Q&A]
I'm looking at migrating from the fabric8 plugin but having difficulty figuring out what equivalent of the `start` goal from `docker-maven-plugin` is for the `kubernetes-maven-plugin`

**Comment by rohanKanojia** [comment]
Umm, are you migrating from docker-maven-plugin -> kubernetes-maven-plugin because you're working with Kubernetes now?

**Comment by mpiggott** [comment]
No, I was trying it because fabric8 was labelled as deprecated with jkube as the replacement

**Reply by rohanKanojia** [reply]
I think there is a misunderstanding. 

https://github.com/fabric8io/fabric8-maven-plugin (plugin to deploy java applications to Kubernetes) is deprecated and https://github.com/eclipse/jkube/tree/master/kubernetes-maven-plugin is the suggested replacement.

https://github.com/fabric8io/docker-maven-plugin is still an active project.

In my opinion, If you use [Docker Maven Plugin](https://github.com/fabric8io/docker-maven-plugin) just for building  / publishing container images, you can switch to Eclipse JKube and leverage on `k8s:build`/ `k8s:push` . However, you want to manipulate docker containers you should stick to Docker Maven Plugin.

**Reply by mpiggott** [reply]
Ah, makes sense I had the impression from the documentation that it was deprecated

**Reply by rohanKanojia** [reply]
We had updated the https://fabric8.io/ website regarding the main Fabric8 Platform getting deprecated but subprojects still continuing. Could  you please provide link to that documentation page?  It's possible we could have missed it.
